pkgs/by-name/kr/krun/package.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, rustPlatform
+, cargo
+, libiconv
+, libkrun
+, makeWrapper
+, rustc
+, passt
+, sommelier
+, mesa
+, opengl-driver ? mesa.drivers
+, withSommelier ? false
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krun";
+  version = "0.0.0-unstable-2024-06-18";
+
+  src = fetchFromGitHub {
+    owner = "slp";
+    repo = finalAttrs.pname;
+    rev = "912afa5c6525b7c8f83dffd65ec4b1425b3f7521";
+    hash = "sha256-rDuxv3UakAemDnj4Nsbpqsykts2IcseuQmDwO24L+u8=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/slp/krun/pull/39.diff";
+      hash = "sha256-CV69L+VDDLRcWgpgDCAYKLlTU9ytFcHhzNgOibWD8KY=";
+    })
+    (fetchpatch {
+      url = "https://github.com/slp/krun/pull/38.diff";
+      hash = "sha256-cK3iDhh+33H16V65lWUXahjmpSxI1HhiLUmkjfkRB7A=";
+    })
+  ];
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    hash = "sha256-efJvEpmSBTrShOc9fwmn2TqC1zYx5WO4LH/6JQtIErA=";
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.cargoSetupHook
+    rustPlatform.cargoBuildHook
+    rustPlatform.cargoInstallHook
+    rustPlatform.bindgenHook
+    cargo
+    rustc
+    makeWrapper
+  ];
+
+  buildInputs = [
+    (libkrun.override {
+      withGpu = true;
+      withNet = true;
+    })
+  ];
+
+  cargoBuildType = "release";
+
+  # Allow for sommelier to be disabled as it can cause problems.
+  wrapArgs = [
+    "--prefix PATH : ${lib.makeBinPath (lib.optional withSommelier [ sommelier ] ++ [ passt ])}"
+  ];
+
+  postFixup = ''
+    wrapProgram $out/bin/krun $wrapArgs \
+      --set-default OPENGL_DRIVER ${opengl-driver}
+  '';
+
+  meta = {
+    description = "krun - run programs from your system in a microVM";
+    homepage = "https://github.com/slp/krunvm";
```

Comment:
same

Reply:
Forgot to update the homepage URL correctly

Reply:
Fixed